external-crates/move/crates/move-vm-types/Cargo.toml

```diff
@@ -18,11 +18,15 @@ bcs.workspace = true
 
 move-core-types.workspace = true
 move-binary-format.workspace = true
-move-vm-profiler.workspace = true
+move-vm-profiler = { workspace = true, features = ["gas-profiler"] }
```

Comment:
Same question here RE: whether this is needed (although maybe I'm failing to grok Cargo features...)

Reply:
welcome to my life